SPECS/ypbind/ypbind.spec

```diff
@@ -2,11 +2,11 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Summary: The NIS daemon which binds NIS clients to an NIS domain
 Name: ypbind
-Version: 2.6.1
-Release: 4%{?dist}
+Version: 2.7.2
+Release: 10%{?dist}
 License: GPLv2
 Source0: https://github.com/thkukuk/ypbind-mt/archive/v%{version}.tar.gz#/ypbind-mt-%{version}.tar.gz
-Url: https://github.com/thkukuk/ypbind-mt
```

Comment:
Same ask for the URL here.

Reply:
The new URL is broken, reverting to GitHub link